snuba/admin/static/tracing/index.tsx

```diff
@@ -298,6 +298,7 @@ function TracingQueries(props: { api: Client }) {
     <div>
       <form>
         <h2>Construct a query</h2>
+        <a href="https://www.notion.so/sentry/Clickhouse-Death-Queries-d3873ff2f55e4b50b515926ba533f794">🛑 WARNING! BEFORE RUNNING QUERIES, READ THIS FIRST 🛑</a>
```

Comment:
I'm assuming the link would be updated to the live docs once #2509 is merged, so we could probably just wait till that is deployed and update the link within this PR.